pkg/cache/cache.go

```diff
@@ -1,3 +1,6 @@
+/*
+Package cache provides a cache that maintains a list for a resource type which it updates as it
```

Comment:
as it ...what?

sorry, missed this before

Reply:
@rmweir you've left us hanging in suspense!

Reply:
I think I was mid comment when I realized I didn't end up making any changes to the cache package 😛 . Removed.